### PR TITLE
Update metadata sync guidelines

### PR DESCRIPTION
**Pull request by calvinjia**
Adds more detail on how to configure metadata sync and the various options available to users.

### CODE REVIEW

File: docs/en/core-services/Unified-Namespace.md [hunk 1]
```diff
@@ -162,22 +162,44 @@ the metadata sync feature is used to synchronize the two namespaces.
...
-If a file is modified in the UFS, Alluxio will detect this from the fingerprint, free the existing
-data for that file, and reload the metadata for the updated file.
-If a file is added or deleted in the UFS, Alluxio will update the metadata in its namespace as well.
+<p align="center">
+<img style="text-align: center" src="{{ '/img/unified-namespace-metadata-sync-performance.png' | relativize_url }}" alt="Metadata sync interval"/>
+</p>
+
+The overall cost of syncing with sync interval enabled (blue line) is amortized across the number of
+RPCs within the sync time interval. When sync interval enabled to some non-0 value users should
+expect performance spikes in normal application usage. The frequency at which the spikes occur will
```
Comment (ZacBlanco) [inline docs/en/core-services/Unified-Namespace.md:188]:
```suggestion
expect latency spikes in normal application usage. The frequency at which the spikes occur will
```

File: docs/en/core-services/Unified-Namespace.md [hunk 2]
```diff
@@ -202,26 +224,133 @@ In `alluxio-site.properties` on master nodes:
...
+of Alluxio, we recommend using the path configuration feature to set the sync interval on the
+specific path(s) based on the understanding of the rate of change in the UFS. This allows files to
+be synced for the special cases without detrimentally affecting the performance of operations on
+other parts of the namespace.
+
+#### UFS frequently updated not through Alluxio
+
+##### HDFS
+
+If the updates happen across many parts (or unknown) of the namespace at a frequent rate, use active
```
Comment (ZacBlanco) [inline docs/en/core-services/Unified-Namespace.md:268]:
```suggestion
If the updates happen across many (or unknown) parts of the namespace at a frequent rate, use active
```

File: docs/en/core-services/Unified-Namespace.md [hunk 3]
```diff
@@ -202,26 +224,133 @@ In `alluxio-site.properties` on master nodes:
...
+We recommend to configure the sync interval exclusively. It is not recommended to consider
+configuring the metadata load type. However, do note the following equivalencies
+
+`alluxio.user.file.metadata.load.type=NEVER` is equivalent to `alluxio.user.metadata.sync.interval=-1`
+`alluxio.user.file.metadata.load.type=ALWAYS` is equivalent to `alluxio.user.metadata.sync.interval=0`
+
+If the metadata sync interval is configured the metadata load type is ignored.
+
+### Active Sync for HDFS
+In version 2.0, we introduced a new feature for maintaining synchronization between Alluxio space
```
Comment (ZacBlanco) [inline docs/en/core-services/Unified-Namespace.md:289]: nit: newline above this line to be consistent?

File: docs/en/core-services/Unified-Namespace.md [hunk 4]
```diff
@@ -202,26 +224,133 @@ In `alluxio-site.properties` on master nodes:
...
+
+#### Quiet period for Active Sync
+
+Active sync also tries to avoid syncing when the target directory is heavily used.
+It tries to look for a quiet period in UFS activity to start syncing between the UFS and the Alluxio space, to avoid overloading the UFS when it is busy.
+There are two configuration options that control this behavior.
+
+`alluxio.master.ufs.active.sync.max.activities` is the maximum number of activities in the UFS directory.
+Activity is a heuristic based on the exponential moving average of number of events in a directory.
+For example, if a directory had 100, 10, 1 event in the past three intervals.
```
Comment (ZacBlanco) [inline docs/en/core-services/Unified-Namespace.md:332]:
```suggestion
For example, if a directory had 100, 10, and 1 events in the past three intervals.
```

Comment (calvinjia) [reply docs/en/core-services/Unified-Namespace.md:332]: Updated, thanks

File: docs/en/core-services/Unified-Namespace.md (hunk 3 above)
Comment (calvinjia) [reply docs/en/core-services/Unified-Namespace.md:290]: Updated, thanks.

File: docs/en/core-services/Unified-Namespace.md (hunk 2 above)
Comment (calvinjia) [reply docs/en/core-services/Unified-Namespace.md]: Updated, thanks.

File: docs/en/core-services/Unified-Namespace.md (hunk 1 above)
Comment (calvinjia) [reply docs/en/core-services/Unified-Namespace.md]: Updated, thanks.

**Comment by calvinjia** [comment]
@ZacBlanco I've updated the image to a smaller one, thanks for catching this.

**Comment by calvinjia** [comment]
alluxio-bot, merge this please.

**Pull request merged**